crypto/asn1/a_mbstr.c

```diff
@@ -272,12 +270,15 @@ static int out_utf8(unsigned long value, void *arg)
 static int type_str(unsigned long value, void *arg)
 {
     unsigned long types;
+    const int native = ossl_fromascii(value);
```

Comment:
> `((c) != (int)(c) ? (c) : ossl_fromascii(c))`

In this case note that implementation-defined signal is an option, and we want to avoid that.